packages/mco/components/protected-applications/components.tsx

```diff
@@ -137,10 +136,7 @@ export const EnrollApplicationButton: React.FC<{ isNoDataMessage?: boolean }> =
                 'pf-u-mt-md': isNoDataMessage,
               })}
             >
-              <OutlinedQuestionCircleIcon
-                size={IconSize.sm}
-                className="pf-u-mr-sm"
-              />
+              <OutlinedQuestionCircleIcon className="pf-u-mr-sm" />
```

Comment:
there are other references in this file too...
```suggestion
              <OutlinedQuestionCircleIcon className="pf-v5-u-mr-sm" />
```